src/ui/actions_main_area.ui

```diff
@@ -265,6 +265,9 @@
            <property name="toolTip">
             <string>&lt;p&gt;Type in one or more commands you want the button to send directly to the game if it is pressed. (Optional)&lt;/p&gt;&lt;p&gt;If this is a &lt;i&gt;push-down&lt;/i&gt; button then this is sent only when the button goes from the &lt;i&gt;up&lt;/i&gt; to &lt;i&gt;down&lt;/i&gt; state.&lt;/p&gt;&lt;p&gt;To send more complex commands, that could depend on or need to modifies variables within this profile a Lua script should be entered &lt;i&gt;instead&lt;/i&gt; in the editor area below.  Anything entered here is, literally, just sent to the game server.&lt;/p&gt;&lt;p&gt;It is permissable to use both this &lt;i&gt;and&lt;/i&gt; a Lua script - this will be sent &lt;b&gt;before&lt;/b&gt; the script is run.&lt;/p&gt;</string>
            </property>
+           <property name="placeholderText">
+            <string>Text to send to the game as-is (optional)</string>
```

Comment:
Only thing is that *this* field one is dual function:
* if it is a normal button then the contents here will be sent ***every*** time the button is clicked
* else (when `checkBox_action_button_isPushDown` is checked) it is only sent ***every other click*** - when the button is pushed down and checked (IIRC).

You will want to duplicate the text you put here when it is **not** a push-down button in: `(void) dlgTriggerEditor::slot_toggle_isPushDownButton(const int state)` and also provide the other text for the push-down button case  in the other `if` branch in that method.

Reply:
I've considered it but I don't think duplicating the label and the tooltip for the third time in the placeholder is a good idea.